mlflow/server/handlers.py

```diff
@@ -3000,28 +3000,49 @@ def _link_traces_to_run():
 @_disable_if_artifacts_only
 def get_trace_artifact_handler():
     request_id = request.args.get("request_id")
+    path = request.args.get("path")
```

Comment:
The backend update is straightforward. We just need to add a `path` to fetch attachments.